priv/repo/migrations/20240724173817_add_user_permissions_column.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Orbit.Repo.Migrations.AddUserPermissionsColumn do
+  use Ecto.Migration
+
+  def change do
+    execute(
+      "CREATE TYPE user_permission AS ENUM ('operator_sign_in')",
+      "DROP TYPE user_permission"
+    )
+
+    alter table("users") do
+      add :permissions, {:array, :user_permission}
```

Comment:
Is this NOT NULL (in the sql sense) by default? i.e. is NULL vs empty array something you're aware of and is handled?

Reply:
Good catch - I thought it usually would default to not null, but this is adding a new column so the default behavior is probably different. I made it non-nullable and added a default value of an empty array in [].

Reply:
3087d2a